<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProyectoConfiguracionesDialog</class>
 <widget class="QewTreeViewDialog" name="ProyectoConfiguracionesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ewTreeViewDialog</string>
  </property>
  <property name="containerName">
   <cstring>stackedWidget</cstring>
  </property>
  <property name="treeNavigatorName">
   <cstring>qewListViewConf</cstring>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="qewListViewConf">
      <column>
       <property name="text">
        <string>Configuraciones</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>Nueva Configuración</string>
       </property>
      </item>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ProyectoConfiguraciones">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="idConfLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Descripción:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="descripcionTextEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QewExtensibleDialog</class>
   <extends>QDialog</extends>
   <header>qewextensibledialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QewTreeViewDialog</class>
   <extends>QewExtensibleDialog</extends>
   <header>qewtreeviewdialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qewListViewConf</sender>
   <signal>itemPressed(QTreeWidgetItem*,int)</signal>
   <receiver>ProyectoConfiguracionesDialog</receiver>
   <slot>setCurrentWidget(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
